src/main/java/gregtech/loaders/recipe/handlers/PartsRecipeHandler.java

```diff
@@ -434,6 +433,9 @@ public static void processTurbine(OrePrefix toolPrefix, Material material, Ingot
         //noinspection ConstantConditions
         TurbineRotorBehavior.getInstanceFor(rotorStack).setPartMaterial(rotorStack, material);
 
+        if (material.hasProperty(PropertyKey.GEM))
```

Comment:
This should not be needed any more after my recent commit which blocks gem Turbine Blades